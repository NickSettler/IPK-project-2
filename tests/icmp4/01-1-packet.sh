./ipk-sniffer -i en0 --icmp4