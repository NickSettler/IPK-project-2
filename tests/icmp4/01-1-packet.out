timestamp: 2023-04-17T20:02:49.254+02:00
src MAC: b0:be:83:4a:c4:b0
dst MAC: 94:08:53:2e:50:cb
frame length: 98 bytes
src IP: 192.168.1.17
dst IP: 192.168.1.49

0x0000: 94 08 53 2e 50 cb b0 be 83 4a c4 b0 08 00 45 00  ..S.P....J....E.
0x0010: 00 54 14 4c 00 00 40 01 e2 ca c0 a8 01 11 c0 a8  .T.L..@.........
0x0020: 01 31 08 00 17 a9 26 ce 00 01 64 3d 89 c9 00 03  .1....&...d=....
0x0030: e0 7a 08 09 0a 0b 0c 0d 0e 0f 10 11 12 13 14 15  .z..............
0x0040: 16 17 18 19 1a 1b 1c 1d 1e 1f 20 21 22 23 24 25  .......... !"#$%
0x0050: 26 27 28 29 2a 2b 2c 2d 2e 2f 30 31 32 33 34 35  &'()*+,-./012345
0x0060: 36 37                                            67

